lib/src/slices/data_bkg/data_bkg_service_email.dart

```diff
@@ -369,4 +369,14 @@ revolution today.<br />
     return apiOauthInterface != null &&
         await apiOauthInterface.isConnected(account);
   }
+
+  Future<void> deleteAccountData(ApiOAuthModelAccount account) async {
+    List<ApiEmailMsgModel> messages =
```

Comment:
other than deleting the corresponding apiAppDataService keys, this logic belongs in the data screen service

Reply:
the data screen service doesn't use the ApiEmailMsgService and ApiEmailSenderService... that's why I have implemented it here...